### PR TITLE
feat(bookmarks): Updates for launching.

### PR DESCRIPTION
**Pull request by blaino**

### CODE REVIEW

**Comment by blaino** [comment]
@arw180 and another one.

File: app/js/components/WebtopLaunchLink.jsx [hunk 1]
```diff
@@ -5,9 +5,7 @@ var React = require('react');
 var { WEBTOP_URL } = require('../OzoneConfig');
 
 function getLink(listing) {
-    var uuid = listing.uuid;
-
-    return `${WEBTOP_URL}#/launchApp/${encodeURIComponent(uuid)}`;
+    return listing.launch_url;
```
Comment (arw180) [inline app/js/components/WebtopLaunchLink.jsx]: So this functionality is a little special - unlike the 'launch in new tab' functionality, the 'launch in webtop' mechanism should actually return the link that was here before. Your change to TabLaunch.jsx is good, but this one should be discarded.

Comment (blaino) [inline app/js/components/WebtopLaunchLink.jsx]: Oops. Yeah, I need to change uuid to unique_name, though.

**Comment by arw180** [comment]
:+1:

**Pull request merged**